POC/ColorPicker/ColorPicker/ColorPickingFunctionality/InternalSystemCallException.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+
+namespace ColorPicker.ColorPickingFunctionality
+{ 
+    class InternalSystemCallException : Exception
```

Comment:
Just to check: there wasn't a built-in exception for this use case? In any case, I think these guidelines should be used when creating a user-defined exception: https://docs.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-user-defined-exceptions

Reply:
Discussed offline - couldn't find a good built in exception which fit.